pyEpiabm/pyEpiabm/person.py

```diff
@@ -84,3 +85,15 @@ def update_status(self,
         self.microcell.notify_person_status_change(
             self.infection_status, new_status)
         self.infection_status = new_status
+
+    def update_time_to_status_change(self) -> None:
+        """Method that assigns time until next infection status update,
+         given as a random integer between 1 and 10.
+
+        :return: Time until next infection status update
+        :rtype: int
```

Comment:
It appears that this function doesn't actually return anything (as declared in L89) - so this is unnecessary

Reply:
Nick was on about having functions to update class params rather than directly changing things